research/object_detection/samples/configs/faster_rcnn_inception_resnet_v2_atrous_oid.config

```diff
@@ -106,6 +106,7 @@ train_config: {
   }
   gradient_clipping_by_norm: 10.0
   fine_tune_checkpoint: "PATH_TO_BE_CONFIGURED/model.ckpt"
+  from_detection_checkpoint: true
```

Comment:
By default we assume the checkpoint used is object classification ckpt. So this may not be necessary.
Please refer to [this](https://github.com/pkulzc/models/blob/master/research/object_detection/g3doc/configuring_jobs.md#model-parameter-initialization) for more details.

Reply:
From what I understand, by setting fine_tune_checkpoint and from_detection_checkpoint helps us use pre-trained model. If from_detection_checkpoint is false, it will pick up what ever available variables from the checkpoint(fine_tune_checkpoint) and use them as initialization for training the model. 
Please correct me if I am wrong.

Reply:
Due to code restructuring, this will not be required now.
Closing the request.